12_glob_lab/globlab.txt

```diff
@@ -4,26 +4,31 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 1. Type in a glob pattern to match:
 
   1. All files whose name starts with a lowercase letter
-  Answer: 
+  Answer: [a-z]*
 
+  Ofir>> for some reason the shell is not case sensitive so also upper case letters are matched, same for the next quesiton. 
+  
   2. All files whose name starts with an uppercase letter
-  Answer:
+  Answer: [A-Z]*
 
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
-  Answer:
+  Answer: *[aeiou].*
 
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
-  Answer:
+  Answer: *[!aeiou].*
 
   5. All files with extension ".c" that are in a folder named "src"
-  Answer:
+  Answer: ls src/*.c
 
   6. All files with extension ".c" under any folder
-  Answer:
+  Answer: ls */*.c
 
 2. Write a command that would delete all files ending with a ".png" under any subdirectory.
+$ rm */*.png
 
 3. The command "ls -l" can take the name of a file or directory and show details on them. From within "wildcards" folder run "ls -l [A-Z]*". What was printed? Why?
+Suppose to print the content of Doc directory. as the match should start with upper case letter. Like I mentioned above, for some reason the match is not case sensitive.
```

Comment:
yeah good catch! I try to [explain here](https://github.com/tocodeil/unix1-course-examples/issues/15) let me know if that makes sense to you..